test/operance_data_row_test.dart

```diff
@@ -0,0 +1,180 @@
+// 🐦 Flutter imports:
+import 'package:flutter/material.dart';
+
+// 📦 Package imports:
+import 'package:flutter_test/flutter_test.dart';
+
+// 🌎 Project imports:
+import 'package:operance_datatable/src/operance_data_column.dart';
+import 'package:operance_datatable/src/operance_data_decoration.dart';
+import 'package:operance_datatable/src/operance_data_row.dart';
+
+void main() {
+  group('Given an OperanceDataRow instance', () {
+    final columns = <OperanceDataColumn<String>>[
+      OperanceDataColumn<String>(
+        name: 'name',
+        columnHeader: const Text('Name'),
+        cellBuilder: (context, item) => Text(item),
+      ),
+    ];
+
+    group('When all required parameters are provided', () {
+      testWidgets(
+        'Then it should create an instance with default values',
+        (tester) async {
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Scaffold(
+                body: OperanceDataRow<String>(
+                  columnOrder: [0],
+                  columns: columns,
+                  row: 'Test Row',
+                  index: 0,
+                  tableWidth: 500.0,
+                ),
+              ),
+            ),
+          );
+
+          expect(find.text('Test Row'), findsOneWidget);
+        },
+      );
+    });
+
+    group('When optional parameters are provided and row is expanded', () {
+      testWidgets(
+        'Then it should create an instance with the provided values',
```

Comment:
Same here could be more specific
```suggestion
  testWidgets(
      'Then it should display the row with correct expansion, checkbox, and interaction behavior',
      (tester) async {
```